microprofile-fault-tolerance/src/test/java/org/wildfly/quickstarts/microprofile/faulttolerance/MicroProfileFaultToleranceITest.java

```diff
@@ -33,10 +33,11 @@
 import jakarta.ws.rs.core.GenericType;
 import jakarta.ws.rs.core.Response;
 import java.net.URL;
+import java.util.ArrayList;
 import java.util.List;
 
 @RunWith(Arquillian.class)
-public class MicroProfileFaultToleranceIT {
+public class MicroProfileFaultToleranceITest {
```

Comment:
The rename here is unnecessary and would make this out of sync with other quickstarts in this category.